com.reprezen.swagedit.openapi3/resources/templates.xml

```diff
@@ -0,0 +1,328 @@
+      responses:
+        '${200}':
+          description: OK
+          schema:
+            $$ref: ${model}</template>
+
+     <template name="put" 
+       id="com.reprezen.swagedit.openapi3.templates.path_put_template" 
+       description="PUT template" 
+       context="com.reprezen.swagedit.openapi3.templates.path_item" 
+       enabled="true">put:
+      summary: Update
+      description: Update the entire list (for collection resources) or an item (for object resources)
+      parameters:
+        - name: ${request}
+          in: body
+          required: true
+          schema:
+            $$ref: ${model}  
+      responses:
+        '${200}':
+          description: OK</template>
+
+     <template name="delete" 
+       id="com.reprezen.swagedit.openapi3.templates.path_delete_template" 
+       description="DELETE template" 
+       context="com.reprezen.swagedit.openapi3.templates.path_item" 
+       enabled="true">delete:
+      summary: Delete 
+      description: Delete the entire list (for collection resources) or an item (for object resources)
+      responses:
+        '${200}':
+          description: OK
+          schema:
+            $$ref: ${model}</template>
+    
+    <template name="patch" 
+       id="com.reprezen.swagedit.openapi3.templates.path_patch_template" 
+       description="PATCH template" 
+       context="com.reprezen.swagedit.openapi3.templates.path_item" 
+       enabled="true">patch:
+      summary: Partial update
+      description: ${Update}
+      parameters:
+        - name: ${request}
+          in: body
+          required: true
+          schema:
+           $$ref: ${model}  
+      responses:
+        '${200}':
+          description: OK
+          schema:
+            $$ref: ${model}</template>
+
+    <template name="options" 
+       id="com.reprezen.swagedit.openapi3.templates.path_options_template" 
+       description="OPTIONS template" 
+       context="com.reprezen.swagedit.openapi3.templates.path_item" 
+       enabled="true">options:
+      summary: Request resource options
+      description: Request information about methods available for resource manipulation 
+      responses:
+        '${200}':
+          description: OK</template>
+
+    <template name="head" 
+       id="com.reprezen.swagedit.openapi3.templates.path_head_template" 
+       description="HEAD template" 
+       context="com.reprezen.swagedit.openapi3.templates.path_item" 
+       enabled="true">head:
+      summary: Query a resource in the server
+      description: HEAD sends request and get the response only in the header, the body should not be used
+      responses:
+        '${200}':
+          description: OK</template>
+          
+<!--
+	RESPONSES
+ -->
+ 
+	<template name="OK response" 
+       id="com.reprezen.swagedit.openapi3.templates.200_response" 
+       description="OK template" 
+       context="com.reprezen.swagedit.openapi3.templates.responses" 
+       enabled="true">2${00}:
```

Comment:
`2${00}:` -> `'2${00}':`
I just double-checked [the spec](https://github.com/OAI/OpenAPI-Specification/blob/OpenAPI.next/versions/3.0.md#patterned-fields-4):
> expected response for that HTTP status code <...> MUST be quoted for compatibility between JSON and YAML (i.e. "200")

We properly quote them in our operation templates (GET, POST, etc), but not in response templates...